packages/xo-web/src/xo-app/proxies/deploy-proxy.js

```diff
@@ -298,13 +298,13 @@ const deployProxy = async ({ proxy } = {}) => {
     try {
       license = await createProxyTrialLicense()
     } catch (error) {
-      alert(
+      await alert(
         _('trialStartButton'),
         <span className='text-danger'>
           <Icon icon='alarm' /> {error.message}
         </span>
       )
-      throw error
+      throw undefined
```

Comment:
This is very far from obvious, add a comment explaining why it's necessary.

Reply:
I was thinking about the right comment.

Reply:
Done :+1: